client/app/bundles/course/assessment/submission/components/answers/index.tsx

```diff
@@ -151,8 +157,13 @@ const SubmissionAnswer = <T extends keyof typeof QuestionType>(
     },
   };
 
+  const liveFeedback = useAppSelector((state) =>
+    getFeedbackByQuestionId(state, questionId),
+  );
+  const isShowingPopup = !!liveFeedback?.isShowingPopup;
+
   return (
-    <>
+    <div style={{ position: 'relative' }}>
```

Comment:
@cysjonathan I am considering to move this CSS to file, but I am not sure the way we organize the CSS then don't know which file should I update. Bellow is all CSS files we have:
```
./theme/github.css
./theme/syntax-highlighting.css
./theme/bouncing-dot.css
./theme/index.css
./theme/sidebar.css
./lib/components/core/fields/AceEditor.css
./lib/components/core/fields/CKEditor.css
```

Reply:
Use `className="relative"`?